DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/vlan/test_vlan_basic_functionality.py

```diff
@@ -0,0 +1,90 @@
+import asyncio
+
+import pytest
+from dent_os_testbed.utils.test_utils.tgen_utils import (tgen_utils_get_dent_devices_with_tgen,
+                                                         tgen_utils_setup_streams, tgen_utils_start_traffic,
+                                                         tgen_utils_stop_traffic, tgen_utils_get_traffic_stats,
+                                                         tgen_utils_get_loss, tgen_utils_dev_groups_from_config,
+                                                         tgen_utils_traffic_generator_connect, )
+from dent_os_testbed.utils.test_utils.br_utils import (configure_bridge_setup, configure_vlan_setup,
+                                                       get_traffic_port_vlan_mapping)
+
+
+port_map = ({"port": 0, "settings": [{"vlan": 1, "untagged": False, "pvid": False}]},
+            {"port": 1, "settings": [{"vlan": 1, "untagged": False, "pvid": False}]},
+            {"port": 2, "settings": [{"vlan": 1, "untagged": False, "pvid": False}]},
+            {"port": 3, "settings": [{"vlan": 2, "untagged": False, "pvid": False}]})
+
+
+@pytest.mark.suite_functional_vlan
+@pytest.mark.asyncio
+@pytest.mark.usefixtures("cleanup_bridges", "cleanup_tgen")
```

Comment:
you should be consistent and use `pytestmark` as in your other tests:
```suggestion
pytestmark = [pytest.mark.suite_functional_vlan,
              pytest.mark.asyncio,
              pytest.mark.usefixtures("cleanup_bridges", "cleanup_tgen")]
```

Reply:
if it will be only one test? Still create `pytestmark`

Reply:
you did this in `test_vlan_changing_default_pvid.py`, so you can do this here as well

Reply:
good point. Did not see the bigger picture